.gitignore

```diff
@@ -122,3 +122,5 @@ venv.bak/
 
 #VS Code
 .vscode/
+
+pre-commit
```

Comment:
why do we need to add pre-commit to gitignore? We use it in our Vagrantfile. So, we definitely need it.

Reply:
I got some pre-commit problem when I committed. Removed now.